templates/dashboard/_cost_tracking_panel.html

```
@@ -0,0 +1,87 @@
+        </span>
+      {% endif %}
+    </div>
+
+    {% if cost_summary.total_cost %}
+      <div class="grid grid-cols-1 md:grid-cols-3 gap-6 mt-4">
+        <div>
+          <div class="text-xs uppercase text-gray-500 dark:text-gray-400">{% translate "LLM spend" %}</div>
+          <div class="text-3xl font-bold text-gray-900 dark:text-gray-100">${{ cost_summary.total_cost|floatformat:2 }}</div>
+          {% if cost_summary.delta_pct is not None %}
+            <div class="text-sm {% if cost_summary.delta_pct > 0 %}text-red-600{% elif cost_summary.delta_pct < 0 %}text-green-600{% else %}text-gray-500{% endif %}">
+              {% if cost_summary.delta_pct > 0 %}+{% endif %}{{ cost_summary.delta_pct|floatformat:1 }}%
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)